src/FirebirdSql.Data.FirebirdClient/Client/Native/FesStatement.cs

```diff
@@ -371,11 +371,13 @@ public override void Execute(int timeout, IDescriptorFiller descriptorFiller)
 		descriptorFiller.Fill(_parameters, 0);
 
 		ClearStatusVector();
-		NativeHelpers.CallIfExists(() =>
-		{
-			_database.FbClient.fb_dsql_set_timeout(_statusVector, ref _handle, (uint)timeout);
-			_database.ProcessStatusVector(_statusVector);
-		});
+		NativeHelpers.CallIfExists(
+			"set_timeout",
```

Comment:
Use `nameof` and the method for better maintainability (all places).

Reply:
Fixed